src/components/list/themes/light/list-item.fluent.scss

```diff
@@ -1,17 +1,18 @@
+@use './list-item.base.scss';
```

Comment:
This should rather be imported and used as the default static styles on the list-header.ts. It shouldn't be imported here.